README.md

```diff
@@ -242,8 +253,8 @@ See and contribute on the [docopts Wiki](https://github.com/docopt/docopts/wiki)
 
 ## Install
 
-You only have to drop the binary, and eventually the `docopts.sh` lib helper, in your PATH.
-The binary is standalone and staticaly linked. So it runs everywhere.
+You only have to drop the binary, and ideally also the `docopts.sh` lib helper, in a directory on your PATH.
```

Comment:
That why I was using `eventually` as it was an helper for my working on examples. But we may introduce the helper now as the main entry point as it change the way of using `docopts`. People may be disagry parsing a comment, which is an old habits I have. And would prefer parse a string variable, which become part of the code.

Reply:
I am trying to translate (perhaps not very well) from [eventuellement)https://en.wiktionary.org/wiki/%C3%A9ventuellement) in French, which can be better translated as _perhaps_ than _eventually_.  The _ideally_ is suggesting that docopts.sh is useful, whereas _optionally_ might be more direct translation in context.  (I speak French reasonably well but I am not a translator :-))

I think quite a lot of people prefer parsing a comment, as it' makes the documentation serve two purposes and keeps the code clean.  I tried the non-comments style and found it less readable because of the extra code required.  However, others may disagree so it's good to have both options.